### ISSUE TITLE
Related to closed issue: unexpected video #1421

### ISSUE THREAD

**Issue by judklaus** [BelledonneCommunications/linphone-android#1501]
Hi @Viish , hope you are doing fine.

The unexpected video error was corrected in issue #1421. This time, I realized that the problem still continues if I start video once and then turn it off.

**Detailed scenario:**
1. Caller initiates an audio call. 
2. Then initiates video and turns it off after a few seconds.
3. Callee pauses the call and then resumes the call --> video call is initiated.

**Both clients have the following video settings:**
Enable video = true
Initiate video calls = false
accept incoming video requests = true

**Logs of both sides:**
https://www.linphone.org:444//tmp/613b0ae5eaf3f_b6b04081fe75941c3a47.gz 
https://www.linphone.org:444//tmp/613b0b028610e_4ae36061c8682b930383.gz

Regards.

**Comment by judklaus** [comment]
Hi @Viish, any update on this issue?

**Comment by Viish** [comment]
Yes I confirm the behavior, but our engineer that will work on it is still in vacations.

**Comment by Viish** [comment]
@AndreaGianarda Can you take a look please?

**Comment by judklaus** [comment]
Hi @Viish are there any updates regarding this issue?

**Comment by judklaus** [comment]
Hi, @Viish, @AndreaGianarda do you have any updates regarding this issue?
